assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -226,4 +226,9 @@ che.openshift.password=devel
 che.openshift.project=eclipse-che
 che.openshift.serviceaccountname=cheserviceaccount
 
+# Which implementation of DockerConnector to use in managing containers. In general,
+# the base implementation of DockerConnector is appropriate, but OpenShiftConnector
+# is necessary for deploying Che on OpenShift. Options:
+#     - 'default'   : Use DockerConnector
+#     - 'openshift' : use OpenShiftConnector
 che.docker_connector.provider=openshift
```

Comment:
@TylerJewell That is a lot better, thank you. I was having trouble coming up with a good name, for some reason.

On this topic, do you think it's important to include this field in `che.env` for the new CLI? I'm inclined to think that this is too specialized and would just add clutter, but I'm not fully familiar with the new CLI and which use cases it's intended for.